libs/pika/executors/tests/unit/std_thread_scheduler.cpp

```diff
@@ -24,7 +24,7 @@ struct custom_type_non_default_constructible_non_copyable
     int x;
     custom_type_non_default_constructible_non_copyable() = delete;
     explicit custom_type_non_default_constructible_non_copyable(int x)
-      : x(x){};
+      : x(x) {};
```

Comment:
I think this will need reverting to satisfy clang-format. Same in `thread_pool_scheduler.cpp`.

Reply:
when I merged, I got clang format errors in code unrelated to mine and I clang-formatted it anyway. I guess I should have ignored it